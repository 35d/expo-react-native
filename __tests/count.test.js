// @flow
import reducer, { INITIAL_STATE } from '../src/Reducers/index';

test('init', () => {
  // $FlowFixMe
  expect(reducer(undefined, { type: '@@INIT' })).toMatchSnapshot();
});

test('INCREMENT_COUNT', () => {
  expect(reducer(INITIAL_STATE, { type: 'INCREMENT_COUNT' })).toMatchSnapshot();
});
