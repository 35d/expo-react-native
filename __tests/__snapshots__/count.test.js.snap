// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`カウント+1 1`] = `
"Snapshot Diff:
- First value
+ Second value

  Object {
    \\"btc\\": 0,
-   \\"count\\": 10,
+   \\"count\\": 11,
    \\"eth\\": 0,
    \\"euro\\": 0,
    \\"isAvailable\\": false,
    \\"ltc\\": 0,
  }"
`;

exports[`カウント-1 1`] = `
"Snapshot Diff:
- First value
+ Second value

  Object {
    \\"btc\\": 0,
-   \\"count\\": 10,
+   \\"count\\": 9,
    \\"eth\\": 0,
    \\"euro\\": 0,
    \\"isAvailable\\": false,
    \\"ltc\\": 0,
  }"
`;

exports[`初期化 1`] = `
"Snapshot Diff:
Compared values have no visual difference."
`;
